applications/polkalokr.md

```diff
@@ -0,0 +1,200 @@
+
+## Team :busts_in_silhouette:
+
+### Team members
+* Imran Ashfaq - CEO / Vision Owner
+* Anis Ahmed Chohan - CTO / Solution Architect
+* Lead Tester to be assigned
+* Mid-Weight Smart contract dev to be assigned
+* Senior Designer: Maccy Adalid
+
+### Contact
+* **Contact Name:** Imran Ashfaq
+* **Contact Email:** imran@polkalokr.com
+* https://polkalokr.com
+
+### Legal Structure 
+* **Registered Address:** 20-22 Wenlock Road, London, England, N1 7GU.
+* **Registered Legal Entity:** Polkalokr Ltd.
+
+### Team's experience
+Our past development work has been subject to NDA and we are unable to share it. However our teams profiles are available on our website and their detailed work history is available via LinkedIn profiles provided below.
+
+### Team Code Repos
+* https://github.com/Polkalokr/Open-Grants-Program
+* https://github.com/Polkalokr/gridlokr-smart-contract
+
+### Team LinkedIn Profiles
+* https://www.linkedin.com/in/imranashfaq/
+* https://www.linkedin.com/in/anischohan/
+* https://www.linkedin.com/in/maccyadalid/
+
+## Development Roadmap :nut_and_bolt: 
+
+### Overview
+* **Total Estimated Duration:** 3 months
+* **Full-time equivalent (FTE):**  2.5
+* **Total Costs:** 18000 DAI
+
+### Milestone 1 Document, Build, Test & Deploy
+* **Estimated Duration:** 3 month
+* **FTE:**  2.5
+* **Costs:** 18000 DAI
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 1. | License | Apache 2.0 |
+| 2. | Documentation |  We will provide both inline documentation of the code and a basic tutorial that explains how a user can execute the code via our front-end.  Users will be provided with transactions IDs that they can look-up to confirm the success |
+| 3. | Testing Guide | The code will have integrated-test coverage (min. 70%) to ensure functionality and robustness. In the guide, we will describe how to run these tests.  The remaining test coverage (~30%) will focus on ensuring front-end integration to the code, via our APIs, is functioning as planned | 
+| 4. | Article/Tutorial | We will release a medium article and tutorials that help users use our service and developers understand how our smart contract works. | 
+| 5. | SmartContract: Gridlokr | We will create a smart contract that will run the following functions:-|
+| 5a. ||CreateLock(addresses[], lockType, toDate, currencyToLock, amount, lockStrategy)|
+| 5b. ||GenerateLockingAddress()|
+| 5c. ||GetTokensAndLock(addresses[], currencyToLock, amount)|
+| 5d. ||ReleaseToken(addresses[], currencyToUnlock, amount)|
+| 5e. ||UpdateRecipientAddress(contractAddress, addresses[]) |  
+| 6. | User Interface | We will design, and develop a simple interface that demonstrates smart contract functionality |
```

Comment:
Could you clarify which smart contract functionality the UI supports (or will it support all of the above features)?  Also do you know which programming language you will use and do you potentially already have some initial designs?